javascript/computer_science/project_knights_travails.md

```diff
@@ -1,41 +1,43 @@
 ### Introduction
 
-Now you're a pro with DFS and BFS.  Let's try using our search algorithms on a real problem.
+Now that you're pro with DFS and BFS. Let's try using our search algorithms on a real problem.
 
 For this project, you'll need to use a data structure that's similar (but not identical) to a binary tree. For a summary of a few different examples, reference [this article](https://www.khanacademy.org/computing/computer-science/algorithms/graph-representation/a/describing-graphs).
 
-A knight in chess can move to any square on the standard 8x8 chess board from any other square on the board, given enough turns (don't believe it?  See [this animation](https://cdn.statically.io/gh/TheOdinProject/curriculum/284f0cdc998be7e4751e29e8458323ad5d320303/ruby_programming/computer_science/project_knights_travails/imgs/00.png)).  Its basic move is two steps forward and one step to the side.  It can face any direction.
+A knight in chess can move to any square on the standard 8x8 chess board from any other square on the board, given enough turns (don't believe it? See [this animation](https://cdn.statically.io/gh/TheOdinProject/curriculum/284f0cdc998be7e4751e29e8458323ad5d320303/ruby_programming/computer_science/project_knights_travails/imgs/00.png)). Its basic move is two steps forward and one step to the side or one step forward and two steps to the side. It can face any direction.
 
-All the possible places you can end up after one move look like this:
+All the possible places you can end up after one move looks like this:
```

Comment:
```suggestion
All the possible places you can end up after one move look like this:
```
The old way was correct. The verb "look" has the subject "places". Because the subject is plural, the verb has no "s"

I realize this is confusing because the singular "move" could seem to potentially be the subject, but it's not. It's part of the "after" prepositional phrase.